EU4toV2/Source/V2World/V2Country.cpp

```diff
@@ -1592,12 +1553,21 @@ void V2Country::newCivConversionMethod(double topTech, int topInsitutions) // ci
 			// at 31 techs behind completely unciv
 			// each institution behind is equivalent to 2 techs behind
 
-			int civLevel = static_cast<int>((totalTechs + 31 - topTech) * 4);
-			civLevel = civLevel + (srcCountry->numEmbracedInstitutions() - topInsitutions) * 8;
+			double civLevel = (totalTechs + 31 - topTech) * 4;
+			civLevel = civLevel + ((double)srcCountry->numEmbracedInstitutions() - (double)topInsitutions) * 8.0;
```

Comment:
Can just put a c++-syle cast outside the parenthesis.

Reply:
done